packages/manager/modules/pci/src/projects/project/notebooks/components/configuration-command/configuration-command.component.js

```diff
@@ -0,0 +1,12 @@
+import controller from './configuration-command.controller';
+import template from './configuration-command.html';
+
+export default {
+  bindings: {
+    notebookSpecs: '<',
+    projectId: '<',
+    subsidiary: '<',
```

Comment:
Maybe we can use `coreConfig.getUser()` to get this but I don't know if we want to inject it or not in the component

Reply:
Yeah... A good idea. I have used `coreConfig.getUser()` inside the component itself